File: Homework_1/CPP/2.cpp
#include <iostream>

int main()
{
    float a,b,c;
    std::cin >> a;
    std::cin >> b;
    std::cin >> c;
    if (a <= b && a <= c)
        std::cout << "Minimum is " << a << std::endl;
    else if(b <= a && b <= c)
        std::cout << "Minimum is " << b << std::endl;
    else
        std::cout << "Minimum is " << c << std::endl;
    return 0;
}

File: Homework_1/JS/2.js
const readline = require('readline-sync');

let val1 = readline.questionInt();
let val2 = readline.questionInt();
let val3 = readline.questionInt();

if (val1 <= val2 && val1 <= val3)
{
  console.log("Minimum is: " + val1);
}
else if (val2 <= val1 && val2 <= val3)
{
  console.log("Minimum is: " + val2);
}
else
{
  console.log("Minimum is: " + val3);
}